src/components/screens/TimetableScreen.js

```diff
@@ -155,15 +160,27 @@ export default class TimetableScreen extends Component {
     return festivals.find(festival => festival.festId.toString() === festId)
   }
 
+  shortenFestName = (festObject, num) => {
+    const shortenedFestName = festObject.festName
+    if (shortenedFestName.length > num && num > 3) {
```

Comment:
remove `&& num > 3` and remove `else if` case. Not needed for what you want to achieve.